### PR TITLE
Added the 'graph-id' option

### PR DESCRIPTION
**Pull request by barreljan**
Added the to remove a specific graph. This can be helpfull with automated clean-ups (eg. use rrdtool to find the 'lastvalue' of nan, then remove the corresponding graph of the found datasource).

### CODE REVIEW

File: cli/remove_graphs.php [hunk 1]
```diff
@@ -43,7 +43,8 @@
 	$host_ids            = array();
 	$graph_template_ids  = array();
 	$regex               = array();
-
+        $graph_ids           = array();
+
```
Comment (netniV) [inline cli/remove_graphs.php]: Spacing is incorrect, use tabs not spaces before the first $

File: cli/remove_graphs.php [hunk 2]
```diff
@@ -63,6 +64,7 @@
 		'graph-type::',
 		'graph-template-id::',
 		'host-template-id::',
+                'graph-id::',
```
Comment (netniV) [inline cli/remove_graphs.php]: Spacing is incorrect, use tabs not spaces before the first $

File: cli/remove_graphs.php [hunk 3]
```diff
@@ -111,6 +113,14 @@
 			$host_template_ids = $value;
 
 			break;
+		case 'graph-id':
+                        if (!is_array($value)) {
```
Comment (netniV) [inline cli/remove_graphs.php]: Spacing is incorrect, use tabs not spaces before the first $

File: cli/remove_graphs.php [hunk 4]
```diff
@@ -159,7 +169,7 @@
 }
 
 if ($list && $force) {
-	print "The --list and --force options are mutually exclusive.  Pick on or the other." . PHP_EOL;
+	print "The --list and --force options are mutually exclusive.  Pick one or the other." . PHP_EOL;
```
Comment (netniV) [inline cli/remove_graphs.php]: good spot

File: cli/remove_graphs.php (hunk 1 above)
Comment (barreljan) [reply cli/remove_graphs.php]: fixed

File: cli/remove_graphs.php (hunk 2 above)
Comment (barreljan) [reply cli/remove_graphs.php]: fixed

File: cli/remove_graphs.php (hunk 3 above)
Comment (barreljan) [reply cli/remove_graphs.php]: fixed

File: cli/remove_graphs.php (hunk 4 above)
Comment (barreljan) [reply cli/remove_graphs.php]: thanks

**Comment by barreljan** [comment]
@netniV All is updated.

**Pull request merged**